src/app/components/MixerCard/MixerCard.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Typography from '../Typography/Typography';
+import style from './MixerCard.module.css';
+
+export default function MixerCard(): JSX.Element {
+  return (
+    <section className={style.cardWrapper}>
+      <div>
+        <Typography size="m" color="dark" children="Moog Matriarch" />
+      </div>
+      <div>
+        <Typography size="m" color="red" children="19/20" />
+      </div>
```

Comment:
Why are these `Typographies` wrapped in extra `div`s, I believe you can achieve the same effect without these.
And those fixed `children` should be props like `title` and `channel`

Reply:
Resolved in commits [319a9e3](https://github.com/Seborider/plugs./pull/30/commits/319a9e35642da28a432cb388858983e973fb91e7) and [e9f7991](https://github.com/Seborider/plugs./pull/30/commits/e9f799108674e7c36759502980fa3c214783dd04)